guide/sharding/README.md

```diff
@@ -1,19 +1,23 @@
 ## Should I shard?
 
-Before you dive in this section, please ask yourself this question. Sharding is only necessary at 2500 guilds - at that point, Discord will not allow your bot to login. With that in mind, you should consider this when your bot is around 2000 guilds, which should be enough time to get this working. Contrary to popular belief, sharding itself is very simple. It can be complex depending on your bot's needs, however. So, if you meet the 2000 guild requirement, please continue with this guide, otherwise, maybe you should wait a bit! Without further ado, let's get started.
+Before you dive in this section, please ask yourself this question. Sharding is only necessary at 2500 guilds—at that point, Discord will not allow your bot to login. With that in mind, you should consider this when your bot is around 2000 guilds, which should be enough time to get this working. Contrary to popular belief, sharding itself is very simple. It can be complex depending on your bot's needs, however. So, if you meet the 2000 guild requirement, please continue with this guide, otherwise, maybe you should wait a bit! Without further ado, let's get started.
```

Comment:
Not related to the change you made, but I feel like the last sentence could be restructured or split into two sentences. I dislike the constant use of commas; it's not very smooth when you read it.
On that same sentence, the "2000 guild requirement" part is a false statement; it's more of a suggestion or a recommendation, but not a requirement.